terraform/cp4ba/main.tf

```diff
@@ -52,7 +55,11 @@ data "ibm_container_cluster_config" "cluster_config" {
 
 # --------------- PROVISION DB2  ------------------
 module "install_db2" {
-  source = "git::https://github.com/ibm-hcbt/terraform-ibm-cloud-pak/blob/main/modules/Db2"
+  source = "git::https://github.com/ibm-hcbt/terraform-ibm-cloud-pak/tree/joel_cloud_pak_terraform_issue220/modules/Db2"
```

Comment:
This should be `source = "github.com/ibm-hcbt/terraform-ibm-cloud-pak.git//modules/db2?ref=terraform-0.13"`

Reply:
Updated reference: source = "git::https://github.com/ibm-hcbt/terraform-ibm-cloud-pak/tree/terraform-0.13/modules/Db2"

Ok, I will merge it on the terraform-0.13 branch